firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -49,42 +49,55 @@ void disableMotors()
 // Drives the motor corresponding to "motor" to it's home position (position 0)
 void homeAxis(int motor[])
 {
+  int * currentMotorPos;
+  int currentMotorDir, currentMotorNumEighthSteps;
   int i;
 
+  currentMotorPos = (motor == xMotor) ? &currentX : &currentY;
+
   // Loop until endstop collision, then fine tune it
   // LEFT and DOWN have same value, as do RIGHT and UP. Use LEFT
   // and RIGHT arbitrarily while tuning end stop for both x and y axes.
 
+  // Sets scale and direction for motor and current position
   digitalWrite(motor[DIR_PIN], LEFT);
```

Comment:
While you're here, can you add this comment above this line:
// Moving motor towards home for rough estimate